drivers/frequency/ad9528/ad9528.h

```diff
@@ -529,7 +529,7 @@ int32_t ad9528_spi_write(struct ad9528_dev *dev,
 			 uint32_t reg_data);
 int32_t ad9528_spi_read_n(struct ad9528_dev *dev,
 			  uint32_t reg_addr,
-			  uint32			   _t *reg_data);
+			  uint32_t *reg_data);
```

Comment:
this is something that you introduced in the previous commit, so fix it there and won't be needed in this commit